docs/usage.rst

```diff
@@ -61,3 +65,64 @@ Example of usage:
     @pytest.mark.limit_memory("24 MB")
     def test_foobar():
         pass  # do some stuff that allocates memory
+
+
+``limit_leaks``
+---------------
+
+.. py:function:: limit_leaks(location_limit: str, filtering_fn: Callable['LeaksFilteringFunction', bool]=None)
```

Comment:
I tried quite a lot but I couldn't make this correctly reference our `LeaksFilteringFunction` in `api.rst` and I keep getting:

```
pytest-memray/docs/usage.rst:75: WARNING: py:class reference target not found: LeaksFilteringFunction
```

@pradyunsg do you know what can be going on?

Reply:
It's generating `pytest_memray.LeaksFilteringFunction` as a `py:attribute`, and then failing to resolve it as a `py:class`. Sphinx autodoc has never really played nicely with type aliases...

Reply:
I've pushed a fixup commit that makes this work. It's a hack, but it's a hack that we've used in the past on other projects, and it works...